src/app/pages/meditationPlayer/MeditationPlayer.tsx

```diff
@@ -32,9 +33,19 @@ export default function MeditationPlayer({
     isPlaying ? '' : playGong();
   }
 
+  function endMeditation() {
+    const oldSessions = JSON.parse(localStorage.getItem('sessions') || '');
+    const today = new Date();
+    const addCurrentSession = { h: hours, m: minutes, date: today };
```

Comment:
`add` implies that something is done here but it only seems to be a variable
```suggestion
    const currentSession = { h: hours, m: minutes, date: today };
```